packages/strapi-plugin-settings-manager/admin/src/components/OldList/index.js

```diff
@@ -1,16 +1,16 @@
 /**
 *
-* List
+* OldList
```

Comment:
Maybe it could be great to create a `_deprecated` folder into the `components` folder to put the old component for history before deleting them in the future?